cypress/integration/org_campaign_page.spec.ts

```diff
@@ -0,0 +1,23 @@
+describe('/o/[orgId]/campaigns/[campId]', () => {
+    beforeEach(() => {
+        cy.intercept('GET', /\/api\/orgs\/1\/campaigns\/941\/actions$/, { fixture: 'dummyEvents' });
+        cy.intercept('GET', /\/api\/orgs\/1\/campaigns\/941$/, { fixture: 'dummyCampaign' });
+    });
+
+        it('contains a campaign title', () => {
+            cy.visit('/o/1/campaigns/941');
+            cy.get('[data-test="campaign-heading"]').should('be.visible');
+        });
+
+        it('contains campaign information', () => {
+            cy.visit('/o/1/campaigns/941');
+            cy.get('[data-test="campaign-information"]').should('be.visible');
+        });
```

Comment:
I don't think this should need to be a separate test from the previous one. Just have one test to assert that _all_ campaign information is being displayed, both the title and the description in the same `it` block.

Reply:
I agree. Is it a problem that this doesn't check the content just the element, as with the title?

Reply:
I would say that the issue is the same here. With a small difference being that you could envision a scenario where the text is so long that it is truncated (which would cause the test to fail if you're looking for the entire text). But for the test case the easiest way to solve that is the make sure that the dummy text isn't that long.